check_env.py

```diff
@@ -0,0 +1,238 @@
+'''
+Environment check script for Astropy tutorials:
+This script checks to if the python environement in the user's computer is safe for running all or a single tutorial.
+The requirements are present in requirements.json files in the respective tutorial folders.
+
+To run this script type,
+
+$_ python check_env.py -n <tutorial_name>
+
+or
+
+$_ python prepare_deploy.py check -n <tutorial_name>
+
+
+If tutorial name is not provided the environement for all tutorials will be checked
+
+'''
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+# Standard library
+import os
+import json
+import inspect
+import sys
+import types
+import six
+
+from astropy import log as logger
```

Comment:
This should be wrapped in a try / except ImportError, with a message that astropy is missing and needs to be installed.  It's probably expected that anyone running astropy tutorials would have some version of astropy installed, but for beginners you never know.  They should get a friendly message if not.  :smile: